examples/config/condor/50-scitokens-credmon.conf

```diff
@@ -4,8 +4,8 @@
 #
 # DAEMON_LIST = $(DAEMON_LIST), CREDD, SEC_CREDENTIAL_MONITOR
```

Comment:
Since this is an example, if the line is mandatory anyway, you might as well have it uncommented.
```suggestion
DAEMON_LIST = $(DAEMON_LIST), CREDD, SEC_CREDENTIAL_MONITOR
```
Same for some of the other mandatory knobs below.

Reply:
`DAEMON_LIST` is fine to uncomment, but I'm not sure we want the local credmon mode enabled, too, "out of the box"

Reply:
Similarly, setting encryption to required without the admin's explicit acknowledgement seems like a disaster waiting to happen 😬 so let's just stick with modifying `DAEMON_LIST`.